src/linkPanel/linkPanel.ts

```diff
@@ -0,0 +1,51 @@
+import { JSONObject } from '@lumino/coreutils';
+import { BoxPanel } from '@lumino/widgets';
+
+import { IGlueSessionSharedModel } from '../types';
+
+/**
+ * The link editor widget.
+ */
+export class LinkWidget extends BoxPanel {
+  constructor(options: LinkWidget.IOptions) {
+    super({ ...options, direction: 'left-to-right' });
```

Comment:
I think we could take the `options.sharedModel` out of the `super` call

Reply:
fixed